### ISSUE TITLE
iocpreactor needs to support TCP half-close

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10694]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| PenguinOfDoom reported|
|-|-|
|Trac ID|trac#1038|
|Type|enhancement|
|Created|2005-05-14 11:08:06Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1038 1038
type__enhancement enhancement
reporter__PenguinOfDoom PenguinOfDoom
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1116068886000000 1116068886000000
changetime__1212331146000000 1212331146000000
version__ 
owner__ 
cc__justinj cc__PenguinOfDoom
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|PenguinOfDoom commented|
|-|-|

```
#!html
<pre>
iocpreactor needs to support closing read or write side of a TCP connection. See
t.i.interfaces.IHalfCloseableDescriptor and .IHalfCloseableProtocol
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|PenguinOfDoom set status to `closed`|
|-|-|

Implemented in [#1760](https://github.com/twisted/twisted/issues/1760)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
